source/retryable-reads/retryable-reads.rst

```diff
@@ -446,8 +446,8 @@ Command Monitoring
 
 `As with retryable writes
 <https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#command-monitoring>`__,
-in accordance with the `Command Monitoring
-<https://github.com/mongodb/specifications/blob/master/source/command-monitoring/command-monitoring.rst>`__
```

Comment:
would you mind updating the text following this, so that it applies to both events and log messages? I missed this before. you could take inspiration from the section on this in [CLAM](https://github.com/mongodb/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#guarantees)